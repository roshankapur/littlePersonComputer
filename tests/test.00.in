1009
1010
9999
2009
3010
3110
2112
1112
4300
-99999
1
2