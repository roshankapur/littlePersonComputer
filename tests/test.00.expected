***Start LPC Execution***
***FATAL ERROR: Illegal opCode***
***LPC execution abnormally terminated***

Registers: 
accumulator +0000
program counter 00002
op-code 00099
operand 00099

      0    1    2    3    4    5    6    7    8    9 
 0 1009 1010 9999 2009 3010 3110 2112 1112 4300 0001 
10 0002 0000 0000 0000 0000 0000 0000 0000 0000 0000 
20 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
30 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
40 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
50 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
60 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
70 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
80 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
90 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
***End LPC Execution***
